### DISCUSSION TITLE
Docs "Your Second App" get_build_props -> get_build_state

### DISCUSSION THREAD

**Discussion by MoonKraken** [framesurge/perseus#147] [Q&A] [answered]
In the "Your Second App" section of the docs there is a code sample with a function

```
#[perseus::autoserde(build_state)]
pub async fn get_build_state(
    _path: String,
    _locale: String,
) -> RenderFnResultWithCause<IndexPageState> {
    Ok(IndexPageState {
        greeting: "Hello World!".to_string(),
    })
}
```

But in the description below it is referred to as `get_build_props()`. Just wanted to see if this was a mistake.

**Comment by arctic-hen7** [comment, marked as answer]
Sorry, yes that is a mistake! NextJS refers to this function with `props` rather than `state`, so I was probably on autopilot. I'll fix that!
